src/utils/graphql/queries/getApplicationsList.query.ts

```diff
@@ -28,6 +28,13 @@ export default gql`
         status
         outcome
         lastActiveDate
+        numberOfAssignedNotStartedReviews
+        numberOfAssignedReviews
+        numberOfChangesRequestedReviews
+        numberOfDraftReviews
+        numberOfReviewsSelfAssignedBySomeoneElse
```

Comment:
I think this field name could be simplified with something as `numberOfAssignedToAnother`